.gitignore

```diff
@@ -18,4 +18,19 @@ yarn-debug.log*
 yarn-error.log*
 
 # broadcasts
-broadcast/*
\ No newline at end of file
+broadcast/*
+
+# Rust
+# will have compiled files and executables
+debug/
+target/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
+Cargo.lock
+
+# These are backup files generated by rustfmt
+**/*.rs.bk
+
+# MSVC Windows builds of rustc generate these, which store debugging information
+*.pdb
```

Comment:
added rust related items to gitignore, but left the original yarn items for backwards compatibility.